std_msgs/Header header
float32 rope_force
float32 rope_position
float32 rope_velocity
